### ISSUE TITLE
Remove untagged images

### ISSUE THREAD

**Issue by skoch123** [garethr/garethr-docker#633]
Hello!
How to remove untagged images from docker?

**Comment by scotty-c** [comment]
I don't believe the module supports removing untagged images, but you can run the following `docker rmi $(docker images | grep "^<none>" | awk "{print $3}")`

**Comment by dbason** [comment]
If you're using the latest version of docker you can just do:
```
docker image prune
```

**Comment by jcrsilva** [comment]
You can set this to run on a cron: https://github.com/spotify/docker-gc

Not exactly the same but gets the job done

**Comment by gdubicki** [comment]
This PR will help to keep only a single tag of a given image: https://github.com/garethr/garethr-docker/pull/696
